app/classes/Transvision/ShowResults.php

```diff
@@ -397,6 +412,10 @@ public static function resultsTable($search_object, $search_results, $page)
             $anchor_name = str_replace(['/', ':'], '_', $key);
 
             $clipboard_target_string  = 'clip_' . md5($target_string);
+            if ($locale2 == 'sr') {
+                $transliterated_string = self::getTransliteratedString($target_string, 'sr-Cyrl');
+                $transliterate_id = substr($clipboard_target_string, 5);
```

Comment:
Ok, got it. 
To make things less confusing, I suggest we rename ids as following:
string_ + md5 for the regular string (instead of clip_, even for the clipboard feature)
transliterated_ + md5 for the transliterated string. 

Then you can use `$transliterate_id = 'transliterate_' . md5($target_string);` instead of the substr. For the button we can then replace the  id with a custom attribute (data-transliterated-id?) since we’re not using it as a real id but only to store data to build the actual id with JS. 

Also, we should be able to copy the Latin string with the clipboard button when the transliterated string is displayed. To do that, when we toggle divs we would need to update the data-clipboard-target attribute on the clipboard span with the right id (string_ + md5 or transliterated_ + md5)